addons/l10n_fr_pos_cert/models/res_company.py

```diff
@@ -64,6 +64,7 @@ def build_order_info(order):
 
             if not orders:
                 msg_alert = (_('There isn\'t any order flagged for data inalterability yet for the company %s. This mechanism only runs for point of sale orders generated after the installation of the module France - Certification CGI 286 I-3 bis. - POS', self.env.company.name))
+                raise UserError(msg_alert)
```

Comment:
Not sure if raising an error is the right answer here. From the look of it, `msg_alert` was intended to be returned so if you raise something, it'll never be returned and maybe some flow are thus broken.

Can you verify that won't have any negative impact where it's called ?

Reply:
Thanks for your feedback.
I tested it and it won't break any flows. Moreover, if I return the `msg_alert` it will create the pdf, but `msg_alert` is not used in the document and it can be confusing. I can modify the document to show the `msg_alert`, but in this specific case, I think it's better to just show a UserError.